aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -413,6 +413,34 @@ Tensor cummaxmin_backward(const Tensor& grad, const Tensor& input, const Tensor&
   return result.scatter_add_(dim, indices, grad);
 }
 
+static inline int64_t diff_check(const Tensor& self, int64_t n, int64_t dim) {
```

Comment:
Add a oneline comment explaining what this function does and what it's intended to be used for